packages/store-sync/test/utils.ts

```diff
@@ -0,0 +1,26 @@
+import { drizzle } from "drizzle-orm/sql-js";
+import { createPublicClient, http } from "viem";
+import { foundry } from "viem/chains";
+import initSqlJs from "sql.js";
+import mudConfig from "../../../e2e/packages/contracts/mud.config";
+import { createWorld } from "@latticexyz/recs";
+import { resolveConfig } from "@latticexyz/store";
+import { recsStorage } from "../src/recs";
+import { sqliteStorage } from "../src/sqlite";
+import { createStorageAdapter, createStore } from "../src/zustand";
+
+const publicClient = createPublicClient({
+  chain: foundry,
+  transport: http(),
+});
+
+export const tables = resolveConfig(mudConfig).tables;
+
+export const { components, storageAdapter: recsStorageAdapter } = recsStorage({ world: createWorld(), tables });
+
+export const useStore = createStore({ tables });
+export const zustandStorageAdapter = createStorageAdapter({ store: useStore });
+
+const SqlJs = await initSqlJs();
+export const db = drizzle(new SqlJs.Database(), {});
+export const sqliteStorageAdapter = await sqliteStorage({ database: db, publicClient });
```

Comment:
prob fine here since this is test specific but as a general thing, it's good practice to put side effect creating behavior inside a function that gets called when needed, rather than it executing on import

Reply:
Makes sense. I was curious about this, viem seems to follow that pattern for their test utils: https://github.com/wevm/viem/blob/main/test/src/utils.ts#L274

I'll try implement it here